### PR TITLE
Update Recipe_empty.txt

### PR DESCRIPTION
**Pull request by Faalagorn**
* Incorporates fixes from https://theindiestone.com/forums/index.php?/topic/24011-minor-correction-to-item-names-in-txt-files/&tab=comments#comment-273074 and #224
* Fixes to match #251 
* Various other fixes

### CODE REVIEW

File: Recipe_empty.txt [hunk 1]
```diff
@@ -22,13 +22,13 @@ Recipe_EN = {
 	Recipe_Build_Spiked_Plank = "Build Spiked Plank",
 	Recipe_Butcher_Bird = "Butcher Bird",
 	Recipe_Butcher_Rabbit = "Butcher Rabbit",
-	Recipe_Butcher_Small_Animal = "Butcher Small Animal",
+	Recipe_Butcher_Small_Animal = "Butcher Rodent",
```
Comment (nsKpz) [inline Recipe_empty.txt:25]: Would make sense for current game, but we're gonna leave it as "Small Animal" for the future.

File: Recipe_empty.txt [hunk 2]
```diff
@@ -159,29 +159,29 @@ Recipe_EN = {
 	Recipe_Make_Suture_Needle = "Make Suture Needle",
 	Recipe_Make_Suture_Needle_Holder = "Make Suture Needle Holder",
 	Recipe_Make_Tent_Kit = "Make Tent Kit",
-	Recipe_Make_Three-Log_Stack = "Make Three-Log Stack",
+	Recipe_Make_Three-Log_Stack = "Make Three Logs Stack",
```
Comment (nsKpz) [inline Recipe_empty.txt:162]: For this and the other Log Stacks, "X-Log Stack" looks/sounds better to us.

File: Recipe_empty.txt [hunk 3]
```diff
@@ -236,7 +236,7 @@ Recipe_EN = {
 	Recipe_Put_Eggs_in_Carton = "Put Eggs in Carton",
 	Recipe_Put_Potato_Seeds_in_Packet = "Put Potato Seeds in Packet",
 	Recipe_Put_Radish_Seeds_in_Packet = "Put Radish Seeds in Packet",
-	Recipe_Put_Strawberry_Seeds_in_Packet = "Put Strawberry Seeds in Packet",
+	Recipe_Put_Strawberry_Seeds_in_Packet = "Put Strawberries Seeds in Packet",
```
Comment (nsKpz) [inline Recipe_empty.txt:239]: Strawberry Seeds would be correct in this case.

File: Recipe_empty.txt (hunk 1 above)
Comment (Faalagorn) [reply Recipe_empty.txt:25]: Reasoning for this was here: https://theindiestone.com/forums/index.php?/topic/24011-minor-correction-to-item-names-in-txt-files/&tab=comments#comment-272982 but I can change it back if you want :)

File: Recipe_empty.txt (hunk 3 above)
Comment (Faalagorn) [reply Recipe_empty.txt:239]: Similar to above, I matched the item name; I agree maybe it's worth to change the other way around (the item name)? https://github.com/TheIndieStone/ProjectZomboidTranslations/blob/master/Items_empty.txt#L974 If so, I'll send another PR for item name, maybe together with logs if we want it that way, it's just important to keep parity between recipe and item name :)

File: Recipe_empty.txt (hunk 2 above)
Comment (Faalagorn) [reply Recipe_empty.txt:162]: I don't disagree, but I changed it to match the item name, see https://github.com/TheIndieStone/ProjectZomboidTranslations/blob/master/Items_empty.txt#L1025 Also, if there's a hyphen, it hsould be lower-cased letter to match other items

> Commented on the stuff to change back. After that it's golden, thank you

Thanks! I commented back, but if needed I'll try adjust the PRs this week or so!

File: Recipe_empty.txt (hunk 1 above)
Comment (nsKpz) [reply Recipe_empty.txt:25]: Yep, please change back :)

File: Recipe_empty.txt (hunk 2 above)
Comment (nsKpz) [reply Recipe_empty.txt:162]: Good points, thanks! ItemNames were adjusted accordingly :)

File: Recipe_empty.txt (hunk 3 above)
Comment (nsKpz) [reply Recipe_empty.txt:239]: Nice, thanks!
Getting rid of one RJism after the other <3

Item names adjusted with https://github.com/TheIndieStone/ProjectZomboidTranslations/commit/4ad969672536f0a82dca8f68009f1b928c950f06

**Comment by Faalagorn** [comment]
Sorry it took a while, but all seems to be addressed now :)

**Pull request merged**